FirebaseABTesting/Sources/Public/FIRExperimentController.h

```diff
@@ -45,11 +46,15 @@ NS_SWIFT_NAME(ExperimentController)
 ///                       (Timestamps are specified by the number of seconds from 00:00:00 UTC on 1
 ///                       January 1970.).
 /// @param payloads       List of experiment metadata.
+/// @param completionHandler Code to be executed after experiments are updated in the background
+/// thread.
 - (void)updateExperimentsWithServiceOrigin:(NSString *)origin
                                     events:(FIRLifecycleEvents *)events
                                     policy:(ABTExperimentPayload_ExperimentOverflowPolicy)policy
                              lastStartTime:(NSTimeInterval)lastStartTime
-                                  payloads:(NSArray<NSData *> *)payloads;
+                                  payloads:(NSArray<NSData *> *)payloads
+                         completionHandler:
+                             (nullable void (^)(NSError *_Nullable error))completionHandler;
```

Comment:
Definitely note that this is a breaking API change in the release notes.

Reply:
Does the breaking change here mean that this PR should be tagged for Firebase 7 and merged then?

Reply:
A corresponding RC change will need to be made. Can we support the existing method so that the change is not tied with the same RC release?

Reply:
Consider adding a new method with this new signature, and in the old method delegating to the new method with a completion handler of `nil`.

If you want to you can remove and deprecate the old method later but imo it's fine to leave as a convenience.

Reply:
Sounds good.